gu-client/src/lib.rs

```diff
@@ -7,6 +7,8 @@ pub mod sync;
 pub use gu_model as model;
 pub use gu_net::NodeId;
 
+pub use gu_net::rpc::PublicMessage;
+
 #[cfg(feature = "integration_tests")]
 #[cfg(test)]
 mod integration_tests;
```

Comment:
those integration tests fails on mac

Reply:
:rofl: they always fail. you should run then from tests/Makefile only (inside docker container)!

Reply:
I'm running with `--all-features` so thats why it was invoked.